downstream/modules/platform/proc-aap-migration-backup.adoc

```diff
@@ -0,0 +1,20 @@
+[id="aap-migration-backup"]
+
+[role="_abstract"]
+Before migrating your data from {PlatformNameShort} 2.x or earlier, you must back up your data for loss prevention. To backup your data, do the following:
+
+.Procedure
+. Log in to your current deployment project.
+. Run setup.sh to create a backup of your current data/deployment:
++
+For on prem deployments:
++
+-----
+$ ./setup.sh -b
+-----
++
+For OpenShift deployments:
```

Comment:
This is the way to back up for Tower 3.8.7 and prior.  Earlier in this file on line 4, it says we are talking about 2.x or earlier.  

So we might want to include something like:

```
For Openshift Operator installations 2.0+, see these docs for using AutomationControllerBackup and AutomationHubBackup objects. <link>
```

Or, we could just change the title of "For on prem deployments:" --> "For on-prem deployments of 2.x or earlier:"

And "For OpenShift deployments:" --> "For OpenShift deployments prior to 2.0 (non-operator deployments)"

Reply:
Second suggestion made and added a reminder to include a link to the new Backup and Restore doc for installations of OpenShift 2.0 and later. This will get included once the new Backup and Recovery Guide is published.